go.mod

```
@@ -1,6 +1,6 @@
 module github.com/k3s-io/k3s
 
-go 1.19
+go 1.21
```

Upstream is still on `go 1.19` in go.mod, can we leave this as-is?
https://github.com/kubernetes/kubernetes/blob/release-1.26/go.mod#L9

```suggestion
go 1.19
```

---

Upstream is building with go 1.21, and running go mod tidy updates this version as follows anyway:
```
go 1.21

toolchain go1.21.7
```
Setting GOTOOLCHAIN doesn't work for me, but anything other than running go mod tidy with an old go version results in the go mod file being updated.

---

yeah sounds like that's unavoidable :/